### PR TITLE
added a  fact  about Margherita pizza

### PR DESCRIPTION
**Pull request by twister904**
## Description
I've contributed a fact about cheese burst  pizza. It's a simple but interesting fact about the pizza's past.
By adding this, I hope it makes the project more enjoyable for everyone who loves pizza and history.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [✔️ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

DominosPizzaPizzaGIF (2)

### CODE REVIEW

File: pizza-facts-&-trivia.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Some say that the queen’s favorite pizza toppings were mozzarella, tomato, and
...
 11. The Hawaiian pizza, featuring pineapple and ham as its key toppings, was not invented in Hawaii. It was actually created in Canada. Sam Panopoulos, a Greek immigrant to Canada, is credited with making the first Hawaiian pizza in his restaurant, the Satellite Restaurant, in Chatham, Ontario, in 1962. This sweet and savory pizza combination has since become a popular and divisive choice for pizza lovers around the world.
+12. Legend has it that the Margherita pizza, a classic pizza topped with tomatoes, mozzarella cheese, basil, and a drizzle of olive oil, was created in honor of Queen Margherita of Savoy, the Queen of Italy in the late 19th century. The combination of red tomatoes, white mozzarella, and green basil was said to represent the colors of the Italian flag. Whether it's a fact or just a charming tale, the Margherita pizza remains a beloved and timeless classic around the world
```
Comment (CBID2) [inline pizza-facts-&-trivia.md:22]: [suggestion] Interesting as this fact is, it was mentioned in number 5 already, so can you come up with a different one?
```suggestion
12. Legend has it that the Margherita pizza, a classic pizza topped with tomatoes, mozzarella cheese, basil, and a drizzle of olive oil, was created in honor of Queen Margherita of Savoy, the Queen of Italy in the late 19th century. The combination of red tomatoes, white mozzarella, and green basil was said to represent the colors of the Italian flag. Whether it's a fact or just a charming tale, the Margherita pizza remains a beloved and timeless classic around the world
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @twister904. Your PR is off to a great start. It just needs a bit more tweaking. :)

File: pizza-facts-&-trivia.md (hunk 1 above)
Comment (twister904) [reply pizza-facts-&-trivia.md:22]: yes i can add another interesting fact ...
DoThisGIF

Comment (twister904) [reply pizza-facts-&-trivia.md]: please have a look on the new commit i have submitted the unique pizza fact now

**Pull request closed without merging**